src/app/about/about.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component } from '@angular/core';
-import { environment } from '../../environments/environment.prod';
+import { environment } from '../../environments/environment';
```

Comment:
we have 3 files in environment folder:
![image](https://github.com/Johngtka/Biomagnethick-app/assets/11686968/c526d762-8e61-417e-bcfa-b1080094374e)

always use `environment .ts` in any import!